bloom-and-vine/src/Components/instagram-posts/components/InstagramPost.js

```diff
@@ -0,0 +1,25 @@
+import './InstagramPost.css'
+
+export default function InstagramPost(props) {
+    const { data } = props;
+    const user = data.user;
+    const caption = data.caption;
+    const permalink = data.urls.permalink;
+    const mediaURL = data.urls.mediaURL;
+    const id = data.postInfo.id;
+
+    function onClickHandler(e) {
+        e.preventDefault();
+
+        location.href = permalink;
```

Comment:
I am sure if this is a problem with my computer but I am getting an error message "Line 14:9:  Unexpected use of 'location'  no-restricted-globals"